### ISSUE TITLE
problem regarding vgg model weights loading

### ISSUE THREAD

**Issue by shaymush** [marcellacornia/mlnet#2]
Hi,
I'm trying to run your code in test mode and I get an exception error:
Layer weight shape(3,3,640,64) not compatible with provided weight shape (64,3,3,3)
What could be the problem?

**Comment by marcellacornia** [comment]
Hi @shaymush, which backend are you using? MLNet supports Theano only.

**Comment by shaymush** [comment]
I'm using Theano backend...

**Comment by marcellacornia** [comment]
Have you set the dim ordering to "th" in the keras.json file?

**Comment by shaymush** [comment]
You're right! I forgot to change it. 
Now it works.
Thanks!
